app/assets/src/components/utils/propTypes.js

```diff
@@ -0,0 +1,51 @@
+import PropTypes from "prop-types";
```

Comment:
I am not sure if this makes sense in utils for now.... 
I believe it is better to keep this under the "report" view for now, and as we refactor more pages and find commonalities, maybe move it to a different folder. 
We can discuss more about this organization offline.